src/Wallet.tsx

```diff
@@ -12,8 +12,12 @@ import {
   useMediaQuery,
 } from "@mui/material";
 import { type Role } from "./WalletContainer";
+import { MessageType, type Message } from "../clients/Messages";
+import { OwnerClient } from "../clients/OwnerClient";
 
 let myAddress: string = "placholder";
+let mySigningKey: string;
+let myPeer: string; // If I'm Alice, this is Bob. If I'm Bob, this is Alice.
```

Comment:
In the future will Irene also have a wallet UI to run the client? Or will the irene client just run in the background of the UI?

Reply:
Not clear yet. I think from a messaging point of view, a third tab might be the easiest thing.